lizmap/modules/lizmap/lib/Commands/WMTSSeed.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Lizmap\Commands;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WMTSSeed extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('wmts:cache:seed')
+            ->setDescription('Generate cache for a layer')
+            ->setHelp('verbose option will provide more info')
+            ->addArgument('repository', InputArgument::REQUIRED, 'The repository ID')
+            ->addArgument('project', InputArgument::REQUIRED, 'The project name')
+            ->addArgument('layers', InputArgument::REQUIRED, 'Coma separated layers list or \'*\' ')
+            ->addArgument('TileMatrixSet', InputArgument::REQUIRED, 'the TileMatrixSet (CRS) for which you want to generate cache')
+            ->addArgument('TileMatrixMin', InputArgument::REQUIRED, 'the min zoom level to generate')
+            ->addArgument('TileMatrixMax', InputArgument::REQUIRED, 'the max zoom level to generate')
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'will do not generate cache, output statistics')
+            ->addOption('bbox', null, InputOption::VALUE_REQUIRED, 'bounding box to restrict generation (4 coma separated values)')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'force cache generation even if cache exists')
+
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $req = new \jClassicRequest();
+
+        $fakeServer = new \Jelix\FakeServerConf\ApacheMod(\jApp::wwwPath(), '/index.php');
+        $fakeServer->setHttpRequest($req->getServerURI());
+        $tileMatrixMin = $input->getArgument('TileMatrixMin');
+        $tileMatrixMax = $input->getArgument('TileMatrixMax');
+        if (!(filter_var($tileMatrixMin, FILTER_VALIDATE_INT) && filter_var($tileMatrixMax, FILTER_VALIDATE_INT))) {
+            $output->writeln('<error>TileMatrixMin and TileMatrixMax must be of type int</error>');
+
+            return 0;
+        }
+        $WMTSCache = new \Lizmap\CliHelpers\WMTSCache(function ($str) use ($output) {$output->writeln($str); });
```

Comment:
Can you add this check and a test about it ?
```suggestion
        }
        if ($tileMatrixMax < $tileMatrixMin) {
            $output->writeln('<error>TileMatrixMax must be greater or equal to TileMatrixMin</error>');

            return 0;
        }
        $WMTSCache = new \Lizmap\CliHelpers\WMTSCache(function ($str) use ($output) {$output->writeln($str); });
```